### PR TITLE
Feature/custom fonts

### PR DESCRIPTION
**Pull request by SoleCincis**
Implement custom font  Titilium Web :
- downloaded from https://fonts.google.com/specimen/Titillium+Web?selection.family=Titillium+Web
- following the expo way , font must be loaded in `App `before` MainApp` renders
  - https://docs.expo.io/versions/latest/guides/using-custom-fonts/
- then is possible to use font key words across the app
 - added new font family everywhere there is some text

### CODE REVIEW

File: src/components/Link.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const data = [{ url: 'https://google.com', linktext: 'Weiterlesen auf maz-online
 
 const LinkStyle = styled.Text`
   color: ${colors.secondary};
-  font-weight: bold;
+  font-family: titillium-web-bold;
```
Comment (donni106) [inline src/components/Link.js]: please add `font-weight: 600;` here, or is it bold anyway?

File: src/components/Title.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { colors } from '../config';
 
 export const Title = styled.Text`
   color: ${colors.primary};
-  font-weight: bold;
+  font-family: titillium-web-bold;
```
Comment (donni106) [inline src/components/Title.js]: please add `font-weight: 600;` here, or is it bold anyway?

File: src/config/styles/html.js [hunk 3]
```diff
@@ -9,25 +9,28 @@ export const html = {
...
-    lineHeight: normalize(22),
+    fontWeight: '600',
+    lineHeight: normalize(30),
     marginBottom: normalize(24)
   },
   h2: {
     color: colors.darkText,
-    fontWeight: '400',
-    fontSize: normalize(20)
+    fontSize: normalize(20),
```
Comment (donni106) [inline src/config/styles/html.js]: please add `font-family: 'titillium-web-regular';` here

File: src/config/styles/html.js [hunk 4]
```diff
@@ -36,18 +39,20 @@ export const html = {
   },
   h5: {
     color: colors.darkText,
-    fontWeight: '400',
-    fontSize: normalize(16)
+    fontSize: normalize(16),
```
Comment (donni106) [inline src/config/styles/html.js]: please add `font-family: 'titillium-web-regular';` here

File: src/config/styles/html.js [hunk 5]
```diff
@@ -36,18 +39,20 @@ export const html = {
...
+    fontWeight: '400'
   },
   h6: {
     color: colors.darkText,
-    fontWeight: '400',
-    fontSize: normalize(14)
+    fontSize: normalize(14),
```
Comment (donni106) [inline src/config/styles/html.js]: please add `font-family: 'titillium-web-regular';` here

File: src/components/Title.js (hunk 2 above)
Comment (SoleCincis) [reply src/components/Title.js]: it's already bold thanks to  titillium-web-bold

File: src/components/Link.js (hunk 1 above)
Comment (SoleCincis) [reply src/components/Link.js]: it's already bold thanks to titillium-web-bold

File: src/components/Title.js (hunk 2 above)
Comment (donni106) [reply src/components/Title.js]: so we can remove `font-weight: 600;` from everywhere, where we ware using `titillium-web-bold`?

Comment (SoleCincis) [reply src/components/Title.js]: yes, done.

**Pull request merged**